### ISSUE TITLE
Excessive bandwidth usage since 1.9.2

### ISSUE THREAD

**Issue by waterfallm** [graft-project/GraftNetwork#347]
I've updated graftnetwork to 1.9.2 and over the last 10 days i've used almost 2.5 Tb bandwidth.

No unwanted processes and top network consumer is GraftNetwork/graftnoded

Anyone else seeing similar issues?

Ubuntu 18.04 with latest patches applied.

**Comment by waterfallm** [comment]
![Annotation 2019-10-15 193318](https://user-images.githubusercontent.com/9378879/66859178-b2704580-ef82-11e9-941d-fe6b047f4a63.png)

**Comment by waterfallm** [comment]
graftnoded consantly > 2Mb/Sec sent

**Comment by lyarlyarpantsonfyar** [comment]
@waterfallm - there is no point is posting mate. Development has been abandoned. No one will care, no one will answer, because there are no devs left who can actually answer your question. Slava has zero clue about the current architecture, and I doubt he even know C++ - hell, I doubt he even know how to set up a SN by himself. So ... sorry to say, you're wasting your time here

**Comment by waterfallm** [comment]
thats tells me what i need to know. I'll just shut the server down before the bandwidth runs out

**Comment by BKdilse** [comment]
still an issue with v1.10.0.

**Comment by avastar** [comment]
For now, you could cap your outbound bandwidth. Most likely it is other
nodes downloading the blockchain.

**Comment by BKdilse** [comment]
Yeah, that's what I ended up doing.  I did have a look at the limits changes in the code, but I couldn't pinpoint where the issue was.

**Comment by Stardock2018** [comment]
# bandwidth settings
out-peers=32 # This will enable much faster sync and tx awareness; the default 8 is suboptimal nowadays
in-peers=32 # The default is unlimited; we prefer to put a cap on this
limit-rate-up=1048576 # 1048576 kB/s == 1GB/s; a raise from default 2048 kB/s; contribute more to p2p network
limit-rate-down=1048576 # 1048576 kB/s == 1GB/s; a raise from default 8192 kB/s; allow for faster initial sync
